README.md

```diff
@@ -2,12 +2,13 @@
 
 [![Build Status](https://travis-ci.org/ayojs/ayo.svg?style=flat&branch=master)](https://travis-ci.org/ayojs/ayo?branch=master)
 
-_(Note: Ayo.js is forked from [Node.js][]. Currently, a lot of the documentation
```

Comment:
removing the underscores means it won't be italicized; why make that change?

Reply:
I started to think it would have been better with a > but then changed opinion on the fly. Sorry about that.